src/FizzWare.NBuilder/ListBuilderExtensions.cs

```diff
@@ -32,6 +32,17 @@ public static IOperable<T> TheLast<T>(this IListBuilder<T> listBuilder, int amou
             return (IOperable<T>)listBuilderImpl.AddDeclaration(declaration);
         }
 
+        public static IOperable<T> IndexOf<T>(this IListBuilder<T> listBuilder, int index)

```

Comment:
If we're going to do this, let's do it right :) Let's modify the `index` parameter to be `params int[] index` so that you can apply the same logic to multiple indexes at the same time.